src/main/resources/db_scripts/createDB.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE quizowanie;
+USE quizowanie;
+
+CREATE TABLE question (
+	quest_id INT NOT NULL,
```

Comment:
Ale w zwiazku z tym ze bedziemy pozniej korzystac z orm to mysle ze rozsadniejsza moze byc wersja rafala ;)